### PR TITLE
Traducción

### PR DESCRIPTION
**Pull request by abcb1122**
Traducción de actualizaciones en PT y EN

### CODE REVIEW

File: english/src/README.md [hunk 1]
```diff
@@ -1,18 +1,32 @@
...
-On this site you will find information about [@lnp2pBot](https://t.me/lnp2pbot). It is an application to buy and sell bitcoin on Lightning Network with your local currency, without giving your personal data.
+Here you'll find information about [@lnp2pBot](https://t.me/lnp2pbot). This is a Telegram bot for buying and selling bitcoin on the Lightning Network, in a peer-to-peer manner, using your local currency without requiring KYC or providing personal data.
 
 ## Quick Start Guide
 
-1. Download a Lightning Network compatible wallet ([Recomended wallets](./recommended-wallets.md)).
-2. Get a Telegram username.
-3. Start [@lnp2pBot](https://t.me/lnp2pbot)
+1. Download a Lightning Network-compatible wallet ([Recommended Wallets](./recommended-wallets.md)).
+2. Get a Telegram user.
```
Comment (grunch) [inline english/src/README.md:8]: please change back this to `username` which is exactly what the user needs to do

File: english/src/README.md [hunk 2]
```diff
@@ -1,18 +1,32 @@
...
+6. If you don't find a suitable offer, create one and wait for someone to take it.
+Type `/buy` in [@lnp2pBot](https://t.me/lnp2pbot) to [buy Sats](./how-do-i-create-a-buy-order.md) or `/sell` to [sell Sats](./how-do-i-create-a-sell-order.md), according to your preference, and it will guide you through the process. If you want your offer to be published in a specific community rather than the general offer channel, you need to [set a default community](./how-do-i-make-offers-in-a-community.md).
 
-5. Start buying and selling.
-6. Find answers to your questions in the [FAQ](./faq.md) section.
+7. Start buying and selling.
+8. Find answers to your questions in the [FAQ section](./faq.md).
+
+## Recommendations
+- If you have any unanswered questions at [https://lnp2pbot/aprende](https://lnp2pbot.com/aprende/), join the [support group](https://t.me/lnp2pbotHelp) and ask there.
```
Comment (grunch) [inline english/src/README.md:28]: Here we should link the english documentation site https://lnp2pbot.com/learn instead

File: english/src/SUMMARY.md [hunk 3]
```diff
@@ -3,7 +3,7 @@
 [@lnp2pBot](README.md)
 
 - [FAQ](./faq.md)
-  - [What is @lnp2pBot?](./what-is-lnp2pbot.md)
+  - [What is lnp2pBot?](./what-is-lnp2pbot.md)
```
Comment (grunch) [inline english/src/SUMMARY.md:6]: please put back the @lnp2pBot

File: english/src/README.md (hunk 1 above)
Comment (abcb1122) [reply english/src/README.md]: done

File: english/src/README.md (hunk 2 above)
Comment (abcb1122) [reply english/src/README.md]: done

File: english/src/SUMMARY.md (hunk 3 above)
Comment (abcb1122) [reply english/src/SUMMARY.md]: done

**Comment by abcb1122** [comment]
solving reviews

**Review by grunch — APPROVED** [review]
LGTM! thanks for this contribution 🥳

**Pull request merged**